libs/api/src/main/java/de/cyface/api/model/ClassifiedSegment.java

```diff
@@ -0,0 +1,434 @@
+     *                        underlying OSM way id or {@code false} if it's orientated in the opposite direction.
+     * @param geometry        The geometry of this segment in the GeoJSON format, i.e. containing a `type` attribute with
+     *                        `LineString` as value and the `coordinates` attribute with an array containing the nodes of this
+     *                        segment, as loaded from the database.
+     * @param length          The length of this segment in meters.
+     * @param modality        The {@link Modality} this segment was recorded with.
+     * @param vnk             The OSM id of the node where the underlying OSM way of this segment starts.
+     * @param nnk             The OSM id of the node where the underlying OSM way of this segment ends.
+     * @param start           The meter this segment start at compared the full way from `vnk` to `nnk`. I.e. the travel-distance
+     *                        to `vnk`.
+     * @param latestDataPoint The time of the newest data point used to classify this segment.
+     * @param username        The name of the user who uploaded the data of this segment.
+     * @param expectedValue   A mean value from probability theory required to update {@code variance} without requiring
+     *                        previous points.
+     * @param variance        The mathematical variance calculated from calibrated vertical accelerations.
+     * @param quality         The surface quality class calculated for this segment.
+     * @param dataPointCount  The number of sample points used to calculate {@code variance}. This is required to update
+     *                        {@code variance} without requiring previous points.
+     * @param version         The version of the format of this segment entry.
+     */
+    @SuppressWarnings("unused") // Part of the API
+    public ClassifiedSegment(final String oid, final boolean forward, final Geometry geometry, final double length,
+                             final Modality modality, final long vnk,
+                             final long nnk, final int start, final OffsetDateTime latestDataPoint, final String username,
+                             final double expectedValue, final double variance,
+                             final SurfaceQuality quality, final long dataPointCount, final String version) {
+        this.oid = oid;
+        this.forward = forward;
+        this.geometry = geometry;
+        this.length = length;
+        this.modality = modality;
+        this.vnk = vnk;
+        this.nnk = nnk;
+        this.start = start;
+        this.latestDataPoint = latestDataPoint;
+        this.username = username;
+        this.expectedValue = expectedValue;
+        this.variance = variance;
+        this.quality = quality;
+        this.dataPointCount = dataPointCount;
+        this.version = version;
+        Validate.isTrue(version.equals(CURRENT_VERSION));
```

Comment:
I added a version to this class as we store and load these objects from the database and need to make sure the structure is in the supported format.

We do the same in `fs.files` and `deserialized`